docs/ske/10-integrations/10-backstage/08-backstage-development-image.mdx

```diff
@@ -0,0 +1,318 @@
+kind: Deployment
+metadata:
+  labels:
+    app: backstage
+  name: backstage
+  namespace: kratix-platform-system
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: backstage
+  template:
+    metadata:
+      labels:
+        app: backstage
+    spec:
+      serviceAccountName: backstage
+      imagePullSecrets:
+      - name: syntasso-registry
+      containers:
+      - env:
+        - name: AWS_ACCESS_KEY_ID
+          valueFrom:
+            secretKeyRef:
+              # update this to your MinIO or S3 bucket credentials
+              name: minio-credentials
+              key: accessKeyID
+        - name: AWS_SECRET_ACCESS_KEY
+          valueFrom:
+            secretKeyRef:
+              # update this to your MinIO or S3 bucket credentials
+              name: minio-credentials
+              key: secretAccessKey
+        - name: SA_TOKEN
+          valueFrom:
+            secretKeyRef:
+              # leave as is, this secret gets created by the manifest
+              name: backstage-token
+              key: token
+        image: registry.syntasso.io/syntasso/backstage-dev-build:v0.12.0-rc49
+        imagePullPolicy: IfNotPresent
+        name: backstage
+        ports:
+        - containerPort: 7007
+          protocol: TCP
+        resources: {}
+        terminationMessagePath: /dev/termination-log
+        terminationMessagePolicy: File
+        volumeMounts:
+        - mountPath: /config
+          name: config
+      dnsPolicy: ClusterFirst
+      restartPolicy: Always
+      schedulerName: default-scheduler
+      securityContext: {}
+      terminationGracePeriodSeconds: 30
+      volumes:
+      - configMap:
+          defaultMode: 420
+          items:
+          - key: config
+            path: app-config.yaml
+          name: backstage
+        name: config
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: backstage
+  namespace: kratix-platform-system
+spec:
+  ports:
+    - port: 7007
+      protocol: TCP
+  selector:
+    app: backstage
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: backstage
+rules:
+# Modify these rules to limit what Backstage can access
+- apiGroups: ["*"]
+  resources: ["*"]
+  verbs: ["*"]
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: backstage
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: backstage
+subjects:
+- kind: ServiceAccount
+  name: backstage
+  namespace: kratix-platform-system
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: backstage
+  namespace: kratix-platform-system
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: backstage-token
+  namespace: kratix-platform-system
+  annotations:
+    kubernetes.io/service-account.name: backstage
+type: kubernetes.io/service-account-token
+```
+
+Once modified and saved, apply the manifest to your cluster:
+
+```bash
+kubectl apply -f backstage.yaml
+```
+
+## Accessing Backstage
```

Comment:
I think we want defaults to be whatever is in the quick start. Do you know if that is node or ingress?

Reply:
its port-forward